src/main/webapp/js/omniComment.es6

```diff
@@ -412,38 +421,37 @@ function enableEdit(commentIdx) {
 }
 
 function enableComment(commentIdx) {
-    $('#commentBar-' + commentIdx).hide();
-    $('#plainCommentText' + commentIdx).hide();
-    $("div[id='commentTextEdit" + commentIdx + "']").show();
-    $("textarea[id='commentText" + commentIdx + "']").val($('#plainCommentText' + commentIdx).text());
+    $(`#commentBar-${commentIdx}`).hide();
+    $(`#plainCommentText${commentIdx}`).hide();
+    $(`div[id='commentTextEdit${commentIdx}']`).show();
+    $(`textarea[id='commentText${commentIdx}']`).val($(`#plainCommentText${commentIdx}`).text());
 
     if (typeof richTextEditorBuilder !== 'undefined') {
         /* eslint-disable camelcase */ // The property names are determined by external library (tinymce)
-        richTextEditorBuilder.initEditor('#commentText' + commentIdx, {
+        richTextEditorBuilder.initEditor(`#commentText${commentIdx}`, {
             inline: true,
-            fixed_toolbar_container: '#rich-text-toolbar-comment-container-' + commentIdx
+            fixed_toolbar_container: `#rich-text-toolbar-comment-container-${commentIdx}`,
         });
         /* eslint-enable camelcase */
     }
 
-    $("textarea[id='commentText" + commentIdx + "']").focus();
+    $(`textarea[id='commentText${commentIdx}']`).focus();
 }
 
 function disableComment(commentIdx) {
-    $('#commentBar-' + commentIdx).show();
-    $('#plainCommentText' + commentIdx).show();
-    $("div[id='commentTextEdit" + commentIdx + "']").hide();
+    $(`#commentBar-${commentIdx}`).show();
+    $(`#plainCommentText${commentIdx}`).show();
+    $(`div[id='commentTextEdit${commentIdx}']`).hide();
 }
 
-function checkComment(form) {
-    if ($(form).find('[id^=commentedittype]').val() === 'delete') {
-        return true;
-    }
-    var formTextField = $(form).find('.mce-content-body');
-    var editor = tinymce.get(formTextField.attr('id'));
-    if (isBlank($(editor.getContent()).text())) {
-        setStatusMessage("Please enter a valid comment. The comment can't be empty.", StatusType.DANGER);
-        scrollToTop();
-        return false;
+function checkComment(form, event) {
+    if ($(form).find('[id^=commentedittype]').val() !== 'delete') {
+        const formTextField = $(form).find('.mce-content-body');
+        const editor = tinymce.get(formTextField.attr('id'));
+        if (isBlank($(editor.getContent()).text())) {
+            setStatusMessage("Please enter a valid comment. The comment can't be empty.", StatusType.DANGER);
+            scrollToTop();
+            event.preventDefault();
```

Comment:
`return false` also implies `event.stopImmediatePropagation();`

I think this function requires it.